### ISSUE TITLE
False positive for unintended_html_in_doc_comment

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/linter#5055]
Example:

```dart
/// `<` or `>`
foo() {}
```

I believe the angle brackets both are quoted.

<img width="1180" alt="image" src="https://github.com/user-attachments/assets/3c2857f3-af68-4359-8779-3dbdd7680b57">

**Comment by kallentu** [comment]
Made a commit that changed the regex which matches for this lint.
Added a test for this case too, which now passes.
Should be good to close now.
